pwa/src/components/applications/applicationsTable.tsx

```diff
@@ -46,11 +48,25 @@ export default function ApplicationsTable() {
         setDocumentation(res.data.content);
       })
       .catch((err) => {
-        setAlert({message: err, type: 'danger'})
+        setAlert({ message: err, type: "danger" });
         throw new Error("GET Documentation error: " + err);
       });
   };
 
+  const handleDeleteApplication = (id): void => {
+    if (confirm(`Do you want to delete this application? With id ${id}`)) {
```

Comment:
Goes for all `confirm()` messages. I feel as if the addition of `id` in this message is unnecessary.

Firstly because the `id` is something used as an identifier for our logic, not something an end-user should be concerned about. Secondly, in most cases we cannot actually see the `id` we're referring to (we're deleting from a table in which we do not see the `id` so it's not any type of verification that the right one was deleted.

I would opt for removing the _With id ..._  part altogether.